content/en/news/2019/announcing-1.4/_index.md

```diff
@@ -26,6 +26,13 @@ are very excited about this improvement and we are working hard to get it
 ready. We've also added new experimental features that don't require Mixer:
 TCP metrics and Stackdriver metrics.
 
+## Authorization policy model in `beta`
+
+The authorization policy model is now in Beta with the introduction of the
```

Comment:
'The authorization policy model is now in Beta' => 'The authorization feature is now in Beta'